packages/native/types/index.d.ts

```diff
@@ -0,0 +1,48 @@
+// Definitions by: Pat Sissons <https://github.com/patsissons>
+// TypeScript Version: 3.4
+
+import '@emotion/core'
```

Comment:
LMK if this makes sense. I think it's required and is best imported here on the api surface.

Reply:
Actually I'm not sure what this was supposed to do - especially in a definition file, but it doesn't matter as I import `Theme` from `@emotion/core` here now, so it's in here one way or another

Reply:
i think it was just to make sure any ambient types were automatically pulled in, but you're right that since we're pulling in `Theme` now it's already there.